### DISCUSSION TITLE
Why the langauge mode of Volar show as "vue" and not "Vue"?

### DISCUSSION THREAD

**Discussion by geekox86** [vuejs/language-tools#340] [General]
Greetings. Thank you so much for this very helpful extension. Really great job.

Just a silly question really.

<img width="400" alt="image" src="https://user-images.githubusercontent.com/2413487/127533590-46243229-818c-4c24-b193-ac940c50422e.png">

**Comment by johnsoncodehk** [comment]
VSCode show "Vue" for me. 🤔 I think this is VSCode problem, volar have no settings for this.

![image](https://user-images.githubusercontent.com/16279759/127773688-95bf5aed-53e0-4884-b270-f5d57545a339.png)
